diff --git a/src/session.c b/src/session.c
index fce12a0b..70b6776a 100644
--- a/src/session.c
+++ b/src/session.c
@@ -981,7 +981,7 @@ session_init (Session *session)
 {
     SessionPrivate *priv = session_get_instance_private (session);
 
-    priv->log_filename = g_strdup (".xsession-errors");
+    priv->log_filename = g_strdup (".cache/xsession-errors");
     priv->log_mode = LOG_MODE_BACKUP_AND_TRUNCATE;
     priv->to_child_input = -1;
     priv->from_child_output = -1;
