diff --git a/armeenm-nix-master/src/libmain/progress-bar.cc b/armeenm-nix-master/src/libmain/progress-bar.cc
index f4306ab91..75e7bfae3 100644
--- a/src/libmain/progress-bar.cc
+++ b/src/libmain/progress-bar.cc
@@ -85,7 +85,7 @@ private:
 public:
 
     ProgressBar(bool printBuildLogs, bool isTTY)
-        : printBuildLogs(printBuildLogs)
+        : printBuildLogs(printBuildLogs || !isTTY)
         , isTTY(isTTY)
     {
         state_.lock()->active = isTTY;
