diff --git a/src/xmpp/omemo.c b/src/xmpp/omemo.c
index 66567e97..88994562 100644
--- a/src/xmpp/omemo.c
+++ b/src/xmpp/omemo.c
@@ -70,7 +70,7 @@ omemo_devicelist_publish(GList* device_list)
 
     log_debug("[OMEMO] publish device list");
 
-    if (connection_supports(XMPP_FEATURE_PUBSUB_PUBLISH_OPTIONS)) {
+    if (false) {
         stanza_attach_publish_options(ctx, iq, "pubsub#access_model", "open");
     }
 
@@ -143,7 +143,7 @@ omemo_bundle_publish(gboolean first)
     g_list_free(lengths);
     g_list_free(ids);
 
-    if (connection_supports(XMPP_FEATURE_PUBSUB_PUBLISH_OPTIONS)) {
+    if (true) {
         stanza_attach_publish_options_va(ctx, iq,
                                          4, // 2 * number of key-value pairs
                                          "pubsub#persist_items", "true",
